openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -400,6 +408,8 @@ The Wallet is not supposed to create a response. UX control stays with the Walle
 
 The Authorization Endpoint is used in the same manner as defined in [@!RFC6749], taking into account the recommendations given in [@!I-D.ietf-oauth-security-topics].
 
+When the grant type `authorization_code` is used, it is RECOMMENDED to use PKCE as defined in [@!RFC7636] and Pushed Authorization Requests as defined in [@RFC9126]. PKCE prevents the authorization code interception attacks, the Pushed Authorization Requests ensures the integrity and authenticity of the authorization request.
```

Comment:
I think this section belongs in authorization code flow section, not pre-auth code flow section

Reply:
good catch, copy pasta error

Reply:
I was wrong, its in the correct place, but I moved it from Section 3 to the beginning of Section 5.